src/components/tooltip/tooltip-theme.scss

```diff
@@ -1,6 +1,6 @@
-md-tooltip.md-THEME_NAME-theme {
-  color: '{{background-700-contrast}}';
-  .md-content {
-    background-color: '{{background-700}}';
+.md-tooltip.md-THEME_NAME-theme {
+  color: '{{background-A100}}';
```

Comment:
@clshortfuse The result of the tooltip using the colors that I have specified is the same exact color scheme of the tooltips that are already present in the material demos. Changing them back to the `background-700` color doesn't work.